google-cloud-redis/Gemfile

```diff
@@ -0,0 +1,12 @@
+source "https://rubygems.org"
+
+gemspec
+
+gem "gcloud-jsondoc",
+    git: "https://github.com/GoogleCloudPlatform/google-cloud-ruby.git",
+    branch: "gcloud-jsondoc"
+
+# WORKAROUND: builds are having problems since the release of 3.0.0
```

Comment:
We should probably add this additional workaround to new libraries like this one (https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-dlp/Gemfile#L9). 

I don't actually know the history of this, perhaps @blowmage can explain

Reply:
Verion 3.0.0 of [public_suffix](https://rubygems.org/gems/public_suffix) dropped support for Ruby 2.0. We occasionally see the build becoming confused by resolving dependencies for a different Ruby version than it is using.